### ISSUE TITLE
language detection for some Chinese punctuations

### ISSUE THREAD

**Issue by tyleemaker** [nvaccess/nvda#12097]
### Steps to reproduce:
1. Open Firefox
2. Browse the following sites:
(a) https://zh-yue.wikipedia.org/
(b) https://lihkg.com/
(c) https://www.facebook.com/
(d) https://github.com/
3. Type "。", "、",,"【", "】", etc.
4. NVDA cannot recognize it as a Chinese punctuation
### Actual behavior:
NVDA reads them as "ideographic period", "ideographic comma", etc.
### Expected behavior:
No matter what the language attributes are, NVDA should read the punctuation marks well in Chinese since I've selected "Trust voice's language when processing characters and symbols".  The file of symbols.dic should be loaded properly.
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2020.4
#### Windows version:
Windoes 8.1 version 6.3.9600 Build 9600
Windows 10
#### Name and version of other software in use when reproducing the issue:
Mozilla Firefox 85 (64bit)
#### Other information about your system:
Microsoft SAPI, Vocalizer Expressive, Windows OneCore, ESPEAK.  All of them do read wrong.
#### Does the issue still occur after restarting your computer?
Yes, definitely
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2020.3 and earlier work well 
#### If addons are disabled, is your problem still occuring?
Yes, definitely
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes, no changes

**Comment by OzancanKaratas** [comment]
This issue occurs because translations of your local language are missing. Please disable the “Include Unicode Consortium data (including emoji) when processing characters and symbols” option in the Speech categorie.

**Comment by tyleemaker** [comment]
Thanks for your reply.  I've tried to disable the linkage with Unicode
Consortium but I find that the file CLDR.dic stops loading.  All emoji
cannot be shown.  If it's a proper performance, I think that a new
feature should be added to the updates.  Allow user's NVDA load
CLDR.dic within the NVDA locale folder even the user disable "Include
Unicode Consortium data".  At this time, the language of locale folder
should be relied on NVDA language or document language.

2021-02-25 1:59 GMT+08:00, Ozancan Karataş <notifications@github.com>:

**Comment by OzancanKaratas** [comment]
@feerrenrut, this issue and #12086 are the same as #11963. Please close this and #12086.

**Comment by CyrilleB79** [comment]
> @feerrenrut, this issue and #12086 are the same as #11963. Please close this and #12086.

I do not clearly understand this issue.
But as indicated in https://github.com/nvaccess/nvda/issues/12086#issuecomment-833916590, #12086 and #11963 do not seem to be identical issues to me.
Please read carefully #12086 before closing it as duplicate.

**Comment by OzancanKaratas** [comment]
Oh, I apologize to everyone.

@tyleemaker, did you checked your local symbols.dic file? Please copy the English symbols.dic file into your language folder and try it, then report your experiences.

**Comment by CyrilleB79** [comment]
@tyleemaker, what is your NVDA language, or Windows language in case you set NVDA language touser  default.
Also which voice synthesizer are you using and with which language?
Have you the issue when reading text from a webpage (maybe Wikipedia example) or reading written text in an edit field (edit field in GitHub for example) or for both? Do you experience the same issue in Notepad?
